package.json

```diff
@@ -26,13 +27,16 @@
   "dependencies": {
     "@hapi/hoek": "^8.0.2",
     "chalk": "^2.4.2",
+    "fast-safe-stringify": "^2.0.7",
     "json-stringify-safe": "^5.0.1",
-    "moment": "^2.24.0"
+    "moment": "^2.24.0",
```

Comment:
`moment` should be moved to devDependencies, since it's only used in tests.

Reply:
> Does it default to sync or async? I see the default option passed to the sonic constructor is `sync: true`, but the flush method seems to default to async.

It defaults to sync. Flush method is only exposed since using async and not loosing logs would require process hooks to flush. Flush behaviour is not relevant to if you run async or not for writes.